build/v1/types.go

```diff
@@ -618,6 +700,9 @@ type CustomBuildStrategy struct {
 
 	// buildAPIVersion is the requested API version for the Build object serialized and passed to the custom builder
 	BuildAPIVersion string `json:"buildAPIVersion,omitempty" protobuf:"bytes,7,opt,name=buildAPIVersion"`
+
+	// Configs is a list of additional configMaps that will be included in the custom build pod
+	Configs []ConfigSpec `json:"configs,omitempty" protobuf:"bytes,8,rep,name=configs"`
```

Comment:
`ConfigMaps`?

Reply:
I had that debate myself - I'm OK with that convention.